components/emoji_picker/emoji_picker_tabs.tsx

```diff
@@ -70,16 +75,19 @@ export default class EmojiPickerTabs extends PureComponent {
                 pickerStyle = {
                     top: this.props.style.top,
                     bottom: this.props.style.bottom,
-                    right: this.props.rightOffset,
+                    right: this.props?.rightOffset,
                 };
             } else {
                 pickerStyle = {...this.props.style};
             }
 
-            pickerStyle.top = pickerStyle.top ? pickerStyle.top + this.props.topOffset : this.props.topOffset;
+            pickerStyle.top = this.props.topOffset || 0;
+            if (pickerStyle.top && this.props.topOffset) {
+                pickerStyle.top = this.props.topOffset + (pickerStyle.top as number);
+            }
 
-            if (pickerStyle.left) {
-                pickerStyle.left += this.props.leftOffset;
+            if (pickerStyle.left && this.props.leftOffset) {
+                (pickerStyle.left as number) += this.props.leftOffset;
```

Comment:
@alexpjohnson can we take a look at this logic one more time, it seems that it is causing the issue of positioning mentioned above.
Fallback to that would be to just revert this piece of logic we are not able to solve the issue

Reply:
Yup. It was a problem with the initial condition of the if statement. Should be good now